e2e/suite1/specs/panelEdit_queries.spec.ts

```diff
@@ -25,6 +25,14 @@ e2e.scenario({
       expect(rows.length).equals(1);
     });
 
+    e2e().server();
+    e2e()
+      .route({
+        method: 'POST',
+        url: '/api/tsdb/query',
+      })
+      .as('apiPostQuery');
```

Comment:
Interesting. Would it be an idea to have a more generic route for `api/*` calls to reuse in other tests in the future?

Reply:
I like that idea, my thinking is to run this for a while and see if this solves the flakiness before introducing it to other tests.